### PR TITLE
Alias --time to --timeout for 'podman stop'

### LINKED ISSUE #366: [v.low] podman stop: --timeout option: allow abbreviating as --time
`docker stop` accepts a `--time` option; in podman, the option is `--timeout`. I think the latter is a better name, but it might be a good idea to enable `--time` as an alias for command-line compatibility with docker.

### PR DESCRIPTION
**Pull request by mheon**
Fixes #366

### CODE REVIEW

File: cmd/podman/stop.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 var (
 	stopFlags = []cli.Flag{
 		cli.UintFlag{
-			Name:  "timeout, t",
+			Name:  "timeout, time, t",
```
Comment (TomSweeneyRedHat) [inline cmd/podman/stop.go]: bash completions?

Comment (TomSweeneyRedHat) [reply cmd/podman/stop.go]: nvm

Comment (TomSweeneyRedHat) [reply cmd/podman/stop.go]: It should complete to timeout now, which is probably what we want.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**